tasks/rubies.yml

```diff
@@ -24,6 +24,14 @@
   when: detect_default_ruby_version.stdout == '' or
         rvm1_default_ruby_version not in detect_default_ruby_version.stdout
 
+- name: Install bundler if not installed
+  shell: 'gem list | if ! grep "^bundler " ; then {{ rvm1_install_path }}/wrappers/{{ item }}/gem install bundler ; fi'
```

Comment:
Sorry for bringing up already merged code, but... Shouldn't it be like so?

```shell
{{ rvm1_install_path }}/wrappers/{{ item }}/gem list | if ! grep "^bundler " ; then {{ rvm1_install_path }}/wrappers/{{ item }}/gem install bundler ; fi
```

Otherwise what exactly `gem` we are calling?